src/core/server/base.js

```diff
@@ -41,6 +42,18 @@ const errorPageText = {
   500: 'Internal Server Error',
 };
 
+function getNoScriptStyles({ appName }) {
+  const cssPath = path.join(config.get('basePath'), `src/${appName}/noscript.css`);
+  try {
+    return fs.readFileSync(cssPath);
+  } catch (e) {
+    if (e.code !== 'ENOENT') {
```

Comment:
Should we at least log that the noscript file doesn't exist as a `debug()` log call, since we're catching an error?